ipn/ipnlocal/local.go

```diff
@@ -120,6 +120,10 @@ import (
 
 var metricAdvertisedRoutes = usermetric.NewGauge(
 	"tailscaled_advertised_routes", "Number of advertised network routes (e.g. by a subnet router)")
+var metricApprovedRoutes = usermetric.NewGauge(
+	"tailscaled_approved_routes", "Number of approved network routes (e.g. by a subnet router)")
+var metricPrimaryRoutes = usermetric.NewGauge(
+	"tailscaled_primary_routes", "Number of primary network routes (e.g. by a subnet router)")
```

Comment:
This needs more docs. I'd say even 90% of this whole stable-metrics-for-customers project is really just the docs part, defining exactly what the metrics mean.